client/src/reducers/search-reducer/search-actions.tsx

```diff
@@ -0,0 +1,19 @@
+export const productPriceAction = (price: { min: number; max: number }) => ({
```

Comment:
put the store, reducers & actions folders under 'src/redux' folder. 
what are these actions for? setting new values? give them better names ('UPDATE_PRODUCT_KIND')
let's talk about this.